src/main/java/mate/academy/spring/controller/UserController.java

```diff
@@ -22,7 +23,7 @@ public UserController(UserService userService,
     }
 
     @GetMapping("/by-email")
-    public UserResponseDto findByEmail(@RequestParam String email) {
+    public UserResponseDto findByEmail(@RequestParam @Valid String email) {
```

Comment:
```suggestion
    public UserResponseDto findByEmail(@RequestParam String email) {
```
here it won't work, because this is not dto